go.mod

```diff
@@ -35,10 +36,14 @@ require (
 	go.mongodb.org/atlas v0.19.0
 	go.mongodb.org/mongo-driver v1.11.0
 	go.mongodb.org/ops-manager v0.43.0
-	golang.org/x/crypto v0.0.0-20220622213112-05595931fe9d
+	golang.org/x/crypto v0.0.0-20220722155217-630584e8d5aa
 	google.golang.org/api v0.102.0
 	google.golang.org/protobuf v1.28.1
 	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.1
+	k8s.io/api v0.25.3
+	k8s.io/apimachinery v0.25.3
+	k8s.io/client-go v0.25.3
```

Comment:
FYI: I will add a new section in [MongoDB Atlas CLI (Atlas CLI) and MongoDB CLI (mongocli)](https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=100255555) to highlight that we will tag your team to review the dependabot PRs related to these dependencies.